file-formats/tabl/indx-v1.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
The json-schema file seems not to be updated, to the new interface version, yet.